pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -3204,7 +3204,7 @@ protected void internalGetBacklogSizeByMessageId(AsyncResponse asyncResponse,
         future.thenAccept(__ -> {
             getPartitionedTopicMetadataAsync(topicName, authoritative, false)
                     .thenAccept(partitionMetadata -> {
-                        if (!topicName.isPartitioned() && partitionMetadata.partitions > 0) {
+                        if (partitionMetadata.partitions > 0) {
```

Comment:
if the `topicName.isPartitioned()`, we don't need to `getPartitionedTopicMetadataAsync`, `managedLedger.getEstimatedBacklogSize(pos)` directly, right? 


The same as below change